components/style/values/specified/basic_shape.rs

```diff
@@ -756,21 +756,21 @@ impl Parse for BorderRadius {
 
 fn parse_one_set_of_border_values(context: &ParserContext, mut input: &mut Parser)
                                  -> Result<[LengthOrPercentage; 4], ()> {
-    let a = try!(LengthOrPercentage::parse(context, input));
+    let a = try!(LengthOrPercentage::parse_non_negavite(input));
```

Comment:
Almost all the `parse` methods implement the [`Parse`](https://doc.servo.org/style/parser/trait.Parse.html) trait, so they're under the compulsion to take the `context` (they simply ignore it if they don't want the context), whereas `parse_non_negative` is [specific](https://doc.servo.org/style/values/specified/length/enum.LengthOrPercentage.html#method.parse_non_negative) to `LengthOrPercentage`